app/views/seriesDetails.scala.html

```diff
@@ -1,5 +1,6 @@
 @import helper._
 @import viewsapi.FormFunctions._
+@import views.html.partials.{errorMessage, errorSummary, inputDropdown, progressIndicator}
```

Comment:
Are these all used on the series page? I'm surprised to see the progressIndicator here.

Reply:
Oh sorry, progressIndicator is the "Step X of Y" header, rather than the upload progress bar.